docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -76,6 +76,14 @@ Error: old cluster config file exists under my-cluster, please use a different c
 The `my-cluster` directory already exists in the current directory.
 Either use a different cluster name or move the directory.
 
+### At least one WorkerNodeGroupConfiguration must not have NoExecute and/or NoSchedule taints
+
+```
+Error: the cluster config file provided is invalid: at least one WorkerNodeGroupConfiguration must not have NoExecute and/or NoSchedule taints
+```
+At least one schedulable worker node for is required to run cluster administration components.
+Remove NoExecute and/or NoSchedule taints from at least one WorkerNodeGroupConfiguration.
```

Comment:
Can we give minimal examples of valid and invalid configurations?

Reply:
Added section with valid and invalid worker node config